From dc09716ec7f1d392f3734de2e4575e7993fe9525 Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Mon, 28 Mar 2016 11:19:44 -0500
Subject: [PATCH 0995/5242] MLK-12602: Documentation: gpmi-nand: legacy bch
 geometry option for NAND

commit  461f54a0166c52dc88e76530a0b6aba87610aed8 from
https://source.codeaurora.org/external/imx/linux-imx.git

document the new option for legacy bch geometry support.

Conflicts:
	Documentation/devicetree/bindings/mtd/gpmi-nand.txt

Signed-off-by: Han Xu <han.xu@nxp.com>
(cherry picked from commit c1c24ecd24cb808e825eb13a3e3d016c283322cc)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/mtd/gpmi-nand.txt          |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/mtd/gpmi-nand.txt b/Documentation/devicetree/bindings/mtd/gpmi-nand.txt
index 3935883..44fae30 100644
--- a/Documentation/devicetree/bindings/mtd/gpmi-nand.txt
+++ b/Documentation/devicetree/bindings/mtd/gpmi-nand.txt
@@ -52,6 +52,10 @@ Optional properties:
   - nand-ecc-step-size: integer representing the number of data bytes
                        that are covered by a single ECC step. The driver
                        supports 512 and 1024.
+ - fsl,legacy-bch-geometry: Use legacy bch geometry(ECC scheme) that
+                       compatible with 3.10 kernel. Without the property,
+                       software may use ECC strength according to NAND chip
+                       spec, e.g. ONFI standard.
 
 The device tree may optionally contain sub-nodes describing partitions of the
 address space. See partition.txt for more detail.
-- 
1.7.9.5

